Dispatcher/Views/ArticleCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ArticleCell.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 19/04/2023.
+//
+
+import UIKit
+
+//MARK: - ArticleCellDelegate
+
+protocol ArticleCellDelegate {
+    func navigateButtonPressed(_ articleTitle: String)
+    func favoritesButtonPressed(_ articleTitle: String)
+    
+}
+
+//MARK: - rticleCell class
+
+class ArticleCell: UITableViewCell {
+    
+    static let articleCellIdentifier = "articleCell"
+    
+
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var tagLabel: UILabel!
+    @IBOutlet weak var autherLabel: UILabel!
+    @IBOutlet weak var subTitleLabel: UILabel!
+    @IBOutlet weak var dispatchButton: UIButton!
+    @IBOutlet weak var moreTagsLabel: UILabel!
+    @IBOutlet weak var favoritesButton: UIButton!
+    
+    var delegate: ArticleCellDelegate?
+    
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        makeCardShape()
```

Comment:
Let's give the function a more meaningful name so it's easier to understand what's inside it.
Since all of the code lines inside are about corner radius (except for the last one) let's call it "setCornerRadiusForCellElements" or something like that (and put the last code line back in the awakefromNib)